src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/GoogleDataprocDAO.scala

```diff
@@ -49,8 +51,6 @@ case class CallToGoogleApiFailedException(googleProject: GoogleProject, context:
 case class FirewallRuleNotFoundException(googleProject: GoogleProject, firewallRuleName: FirewallRuleName)
   extends LeoException(s"Firewall rule ${firewallRuleName.string} not found in project ${googleProject.value}", StatusCodes.NotFound)
 
-case class AuthorizationError() extends LeoException(s"Your account is unauthorized", StatusCodes.Unauthorized)
```

Comment:
We had 2 `AuthorizationError` case classes. I removed this one in favor of the one in LeonardoService.